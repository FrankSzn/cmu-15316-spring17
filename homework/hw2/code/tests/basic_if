as principal admin password "admin" do   
   set x = if false then "success" else "failure"
   return x
***