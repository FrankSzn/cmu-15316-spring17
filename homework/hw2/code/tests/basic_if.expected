{"status":"SET"}
{"output":"failure","status":"RETURNING"}
